### PR TITLE
Faq section aligned and styled in mobile view

### LINKED ISSUE #390: Aligning and improving the view of FAQ section in mobile view
The FAQ section in the mobile view in not good and needs to get alligned.

Kindly assign this issue to me @harikesh409 @ravikr126

### PR DESCRIPTION
**Pull request by harshb02**
# Description
Faq section aligned and styled in mobile view

Please include a summary of the change and which issue is fixed.
Fixes #390 

## List any dependencies that are required for this change 
Na

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
On my local host

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
   - 
WhatsApp Image 2021-04-19 at 8 52 14 PM

    - After :
    
WhatsApp Image 2021-04-19 at 8 51 29 PM

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by ravikr126** [comment]
REMOVE MERGE CONFLICTS @harshb02

**Comment by harshb02** [comment]
Removed the conflicts @ravikr126

**Review by ravikr126 — APPROVED** [review]

**Review by harikesh409 — COMMENTED** [review]
Aren't these changes same as #429

**Comment by harshb02** [comment]
> Aren't these changes same as #429 

No in #429 I have aligned the Sharpen your Skills sectionin mobile view and in this PR I have aligned the FAQ section in mobile view.

File: index.html [hunk 1]
```diff
@@ -146,6 +146,7 @@
 
     <!-- header ends -->
     <!--social media sidebar starts-->
+    <div class="d-none d-lg-block">
```
Comment (harikesh409) [inline index.html:149]: Correct the indentations.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by harikesh409** [comment]
> > Aren't these changes same as #429
> 
> No in #429 I have aligned the Sharpen your Skills sectionin mobile view and in this PR I have aligned the FAQ section in mobile view.

Okay both the changes are present here so I thought they were the same changes anyways do the requested changes we can merge it.

**Comment by harshb02** [comment]
I have removed the div because it was for not displaying the social sidebar in the mobile view but eventually it was done by someone else so I have removed the div and have done the changes @harikesh409

**Comment by harikesh409** [comment]
> I have removed the div because it was for not displaying the social sidebar in the mobile view but eventually it was done by someone else so I have removed the div and have done the changes @harikesh409

okay resolve the conflicts so that we can merge.

**Comment by ravikr126** [comment]
resolve the conflict @harshb02

**Comment by harshb02** [comment]
Removed the conflicts @ravikr126  @harikesh409

**Review by ravikr126 — APPROVED** [review]

**Pull request merged**